src/main/java/ai/aliz/jarvis/db/BigQueryExecutor.java

```diff
@@ -0,0 +1,242 @@
+package ai.aliz.jarvis.db;
+
+import lombok.AllArgsConstructor;
+import lombok.Lombok;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.BaseJsonNode;
+import com.fasterxml.jackson.databind.node.BooleanNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.NullNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.fasterxml.jackson.databind.node.ValueNode;
+import com.google.api.client.util.Lists;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.FieldList;
+import com.google.cloud.bigquery.FieldValue;
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.Table;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TableResult;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import ai.aliz.jarvis.service.shared.platform.BigQueryService;
+import ai.aliz.jarvis.service.shared.ExecutorServiceWrapper;
+import ai.aliz.jarvis.context.Context;
+import ai.aliz.jarvis.util.JarvisUtil;
+
+import static ai.aliz.jarvis.util.JarvisConstants.TEST_INIT;
+
+@Component
+@AllArgsConstructor
+@Slf4j
+public class BigQueryExecutor implements QueryExecutor {
+    
+    private ExecutorServiceWrapper executorService;
+    
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    
+    @Autowired
+    private BigQueryService bigQueryService;
+    
+    @Override
+    public void executeStatement(String query, Context context) {
+        TableResult queryResult = executeQueryAndGetResult(query, context);
+        log.debug("Query result {}", queryResult.iterateAll());
+    }
+    
+    @Override
+    public String executeQuery(String query, Context context) {
+        TableResult queryResult = executeQueryAndGetResult(query, context);
+        ArrayNode result = bigQueryResultToJsonArrayNode(queryResult);
+        return result.toString();
+    }
+    
+    @Override
+    public void executeScript(String query, Context context) {
+        List<String> deletes = Lists.newArrayList();
+        List<String> inserts = Lists.newArrayList();
+        Arrays.stream(query.split(";"))
+              .map(String::trim)
+              .filter(e -> !e.isEmpty())
+              .map(String::toUpperCase)
```

Comment:
Should not change to upper case.
Fixed on init-action-integration-tests branch.